### PR TITLE
GSVA analysis of proteomics data

### PR DESCRIPTION
**Pull request by JosephDybas**
This commit includes the code to run GSVA analysis on the whole cell proteomics data for the HOPE cohort. The GSVA R library is implemented to run the GSVA enrichment analysis. The Kids First IDs are changed to sample IDs (SG IDs) in order to compare to other data and established cell lines.

### CODE REVIEW

File: analyses/proteomic-data/proteomicsGSVA.R
Comment (jharenza) [inline analyses/proteomic-data/proteomicsGSVA.R:7]: @JosephDybas can you add a new PR to add missing packages to docker please :)

File: analyses/proteomic-data/proteomicsGSVA.R
Comment (zzgeng) [inline analyses/proteomic-data/proteomicsGSVA.R:22]: Probably we can define the directories using file.path, instead of using paste

for example:
data_dir <- file.path(root_dir, "data")

So it will work on different operation systems

File: analyses/proteomic-data/proteomicsGSVA.R
Comment (zzgeng) [inline analyses/proteomic-data/proteomicsGSVA.R:42]: I think this file is output folder

**Comment by jharenza** [comment]
@JosephDybas I think this is ready for another look by you

File: analyses/proteomic-data/proteomicsGSVA.R
Comment (JosephDybas) [reply analyses/proteomic-data/proteomicsGSVA.R:22]: Changed this to use file.path as you suggested.

File: analyses/proteomic-data/proteomicsGSVA.R
Comment (JosephDybas) [reply analyses/proteomic-data/proteomicsGSVA.R:42]: I anticipated that this would be confusing. I'm using the liftover datafile, which were written to the "output" directory when they were created. That directory is where I'm reading the file from, so I have the confusing situation of reading the data into the script from the output directory. I can change the directory names but I don't want to adversely impact others who might be working in this repo with the current directory structure. Happy to do what is best for everyone.

**Comment by JosephDybas** [comment]
> hi @JosephDybas can you merge master/main into this and rerun?

I think merging is blocked until @zzgeng reviews. Thanks!

**Comment by jharenza** [comment]
> > hi @JosephDybas can you merge master/main into this and rerun?
> 
> 
> 
> I think merging is blocked until @zzgeng reviews. Thanks!

Sorry- I didn't mean merge the PR. I meant merge the main branch into this PR, update your code to intake the new files which aren't lifted over, and rerun so it can be re-reviewed. Last step before any merge TO main must be merging main into your branch and if any conflicts/updates required for rerun, then rerun. Let me know if we can help.

**Review by zzgeng — APPROVED** [review]
Code works fine and ressults are reproducible! Approving. 
One minor issue could be that `.RData` file is still shown up even it was added to `.gitignore`. Probably you could address this issue.

**Review by jharenza — COMMENTED** [review]
@JosephDybas can you delete the .RData file please

**Pull request merged**
